operator-pipeline-images/operatorcert/entrypoints/index.py

```diff
@@ -113,15 +119,17 @@ def add_bundle_to_index(
     iib_url: str,
     indices: List[str],
     image_output: str,
+    mode: Optional[str] = None,
```

Comment:
Do we need a default parameter here if we always provide the option?

Reply:
Fixed